README.md

```diff
@@ -226,13 +234,30 @@ spec:
   application:
     apiVersion: apps/v1
     kind:       Deployment
-    name:       online-banking
```

Comment:
I think it may be nice to keep the current simple example that uses `name` and doesn't have env or mappings, and then add the additional example that uses labels, env and mappings.  This way it shows a basic vs advanced sample.

Reply:
for the streamlined sample let's also ax `.spec.name` since it's an optional field. We can bring it back in the full example.

Reply:
Addressed in 81ff43f